docs/getting-started-guides/scratch.md

```diff
@@ -66,21 +67,25 @@ the node is added.  A process in one pod should be able to communicate with
 another pod using the IP of the second pod.  This connectivity can be
 accomplished in two ways:
 
-- Configure network to route Pod IPs
+- **Configure underlay network to route Pod IPs**
   - Harder to setup from scratch.
   - Google Compute Engine ([GCE](/docs/getting-started-guides/gce)) and [AWS](/docs/getting-started-guides/aws) guides use this approach.
-  - Need to make the Pod IPs routable by programming routers, switches, etc.
-  - Can be configured external to Kubernetes, or can implement in the "Routes" interface of a Cloud Provider module.
+  - Need to make the Pod IPs routable by programming routers, switches, etc.  
+    - This can be done in a few different ways:
+      - Implement in the "Routes" interface of a Cloud Provider module.
+      - Manually configure static routing external to Kubernetes.
   - Generally highest performance.
-- Create an Overlay network
+- **Use a network plugin**
```

Comment:
It seems like there are two axes here, getting crammed into one:

1. where support lives
   1. in a script or person who statically programs your network hardware, and has no direct interaction with kubernetes.  You have to write this yourself, but there are no rules.
   1. in a plugin which is a separate piece of software that Kubernetes calls out to, called a Network Plugin.  You can use an existing one or write your own.  You need to go find one somewhere other than on the Kubernetes website.
   1. support compiled directuly into Kubernetes (if you don't see one you like, you'd need to write some go code, and then building your own fork of Kubernetes or send us a PR). 
1. How the packets are made to go to the Pod IPs
  - by distributing routes (== underlay?)
  - by encapsulation (== overlay ?)